### PR TITLE
EPAS 15 - Reorg prototype

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

Branch used for prototyping and eventual publishing of the new role and use-case based structure for the EPAS docs

Replaces https://github.com/EnterpriseDB/docs/pull/4199

### CODE REVIEW

**Comment by drothery-edb** [comment]
Replaced by https://github.com/EnterpriseDB/docs/pull/4201

File: product_docs/docs/epas/15/fundamentals/epas_fundamentals/benefits_epas [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (nidhibhammar) [inline product_docs/docs/epas/15/fundamentals/epas_fundamentals/benefits_epas:1]: Any particular reason why this file doesn't have an extension (.mdx)

File: product_docs/docs/epas/15/epas_security_guide/05_edb_audit_logging/03_enabling_audit_logging.mdx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Enabling audit logging"
+description: "How to configure EDB Postgres Advanced Server to log the items you want to audit'
```
Comment (drothery-edb) [inline product_docs/docs/epas/15/epas_security_guide/05_edb_audit_logging/03_enabling_audit_logging.mdx:3]:
```suggestion
description: "How to configure EDB Postgres Advanced Server to log the items you want to audit"
```

**Comment by josh-heyer** [comment]
I think this covers the bulk of the redirects needed. There are a couple of existing paths that won't map, but they appear to be things that should've been removed anyway and I doubt there's much in the way of outstanding links.

**Review by josh-heyer — APPROVED** [review]
I think this is good to merge - what say you, @francoughlin ?

**Comment by francoughlin** [comment]
> I think this is good to merge - what say you, @francoughlin ?

Agreed, @josh-heyer! Will do that now. Thanks for reviewing!

**Review by francoughlin — APPROVED** [review]

**Pull request merged**